distribution/lib/Standard/Microsoft/0.0.0-dev/src/Microsoft365_Credential.enso

```diff
@@ -19,7 +19,7 @@ type Microsoft365_Credential
         - `credentials`: a Cloud secret or a file containing Microsoft365 credentials.
     initialize : File|Enso_Secret -> Microsoft365_Credential
     initialize credentials = if credentials.is_a Microsoft365_Credential then credentials else
-        credentials_reference = as_credential_reference credentials
+        credentials_reference = as_credential_reference credentials CredentialReference
```

Comment:
Lets remove the default value on `as_credential_reference` and always pass `CredentialReference` as that way we wont break this later